README.md

```diff
@@ -219,3 +222,31 @@ function isMetaEqual(a, b) {
 }
 ```
 
+## Patch Application
+
+When you want to apply changes from one tree to another easily, you can use the
+`FSTree.applyPatch` method. For example, given:
+
+```js
+var patch = oldInputTree.calculatePatch(newInputTree);
+var inputDir = 'src';
+var outputDir = 'dist';
+FSTree.applyPatch(inputDir, outputDir, patch);
+```
+
+It will apply the patch changes to `dist` while using `src` as a reference for
+non-destructive operations (`mkdir`, `create`, `change`). You can optionally
+provide a delegate object to handle applying specific types of operations:
+
```

Comment:
Technically, delegates can be anything in the current implementation. Should we restrict it or simply note what the default types are?